code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -485,6 +492,9 @@
 				if(2)
 					L.receive_damage(0,5)
 					Stun(100)
+			if((L.owner.has_trait(TRAIT_EASYDISMEMBER)) && L.body_zone != BODY_ZONE_CHEST)

```

Comment:
should probably be linked to a limb property rather than the owner, but that'd require some reworking